BLL/Services/CandidateService.cs

```diff
@@ -116,5 +116,18 @@ public IEnumerable<CandidateDTO> FindCandidates(string textSearch)
         {
             return _mapper.Map<IEnumerable<Candidate>, IEnumerable<CandidateDTO>>(_db.Candidates.FindWithSpecificationPattern(new CandidatesSearchByAdmin(textSearch)));
         }
+
+        public IEnumerable<CandidateForTechDTO> GetAllCandidatesWithHrInterview()
+        {
+            return _db.CandidatesSandboxes.FindWithSpecificationPattern(new CandidateForHRSpecification())
+                .Select(x => new CandidateForTechDTO
+                {
+                    Id = x.CandidateID,
+                    FirstName = x.Candidate.FirstName,
+                    LastName = x.Candidate.LastName,
+                    IsInterviewedByHR = x.Candidate.Interviews.Count() > 0,
+                    IsInterviewedByTech = x.Candidate.Interviews.Count() < 1,
```

Comment:
interviews count > 0 - means that HR already did interview
interviews count = 0 - means that TECH already did interview??

Reply:
done